README.md

```diff
@@ -173,16 +173,41 @@ $ cp .env.example .env
 $ vi .env
 
 # if you start wintermute without parameters, it will list all available use cases
-$ python wintermute.py
-usage: wintermute.py [-h] {linux_privesc,minimal_linux_privesc,windows privesc} ...
-wintermute.py: error: the following arguments are required: {linux_privesc,windows privesc}
+$ python src/hackingBuddyGPT/cli/wintermute.py
+usage: wintermute.py [-h]
+                     {LinuxPrivesc,WindowsPrivesc,ExPrivEscLinux,ExPrivEscLinuxTemplated,ExPrivEscLinuxHintFile,ExPrivEscLinuxLSE,MinimalWebTesting,WebTestingWithExplanation,SimpleWebAPITesting,SimpleWebAPIDocumentation}
+                     ...
+wintermute.py: error: the following arguments are required: {LinuxPrivesc,WindowsPrivesc,ExPrivEscLinux,ExPrivEscLinuxTemplated,ExPrivEscLinuxHintFile,ExPrivEscLinuxLSE,MinimalWebTesting,WebTestingWithExplanation,SimpleWebAPITesting,SimpleWebAPIDocumentation}
+```
+
+## Provide a Target Machine over SSH
+
+The next important part is having a machine that we can run our agent against. In our case, the target machine will be situated at `192.168.122.151`.
+
+We are using vulnerable Linux systems running in Virtual Machines for this. Never run this against real systems.
+
+{% callout title="We also provide vulnerable machines!" %}
```

Comment:
I don't believe that this works, callout is a feature of our documentation site template.. I don't think github supports this :-/